spec/java/util/_interfaces.lsl

```diff
@@ -8,7 +8,6 @@ library std
 
 // imports
 
-import java.common;
```

Comment:
Now this is needed to add "import java/util/function/_interfaces;" (as I understand)

In this lsl file we have Consumer;